pcsx2/CDVD/IsoHasher.cpp

```diff
@@ -149,7 +149,7 @@ bool IsoHasher::ComputeTrackHash(Track& track, ProgressCallback* callback)
 	std::vector<u8> sector_buffer(sector_size);
 
 	const u32 update_interval = std::max<u32>(track.sectors / 100u, 1u);
-	callback->SetFormattedStatusText("Computing hash for track %u...", track.number);
+	callback->SetFormattedStatusText(tr("Computing hash for track %u..."), track.number);
```

Comment:
The tr function only works in Qt code (it's a member of QObject). You'll need to use one of the translation macros defined at the bottom of Host.h instead.